### PR TITLE
docs: update doc for conditional no drain during ICSP changes

### PR DESCRIPTION
**Pull request by sinnykumari**

### CODE REVIEW

**Comment by sinnykumari** [comment]
This covers changes made in https://github.com/openshift/machine-config-operator/pull/2596/

File: docs/MachineConfigDaemon.md [hunk 1]
```diff
@@ -157,12 +157,16 @@ The updated list of optimized updates and behaviour (as of Openshift 4.8) is as
 1. [SSH Keys](./Update-SSHKeys.md) (updating ignition/passwd/users/sshAuthorizedKeys section in a MachineConfig)
 2. kube-apiserver-to-kubelet-signer CA cert (located at `/etc/kubernetes/kubelet-ca.crt`, 1 year expiry autorotated by the openshift-kubeapiserver operator)
 3. [Pull Secret](./PullSecret.md) (cluster-wide, located at `/var/lib/kubelet/config.json`).
+4. **Selected registries.conf changes(/etc/containers/registries.conf, e.g. ICSP changes)** It can only include following changes in registries.conf file:
```
Comment (yuqi-zhang) [inline docs/MachineConfigDaemon.md:160]: Maybe something like: `Selected registries.conf changes(/etc/containers/registries.conf, e.g. ICSP changes)` -> `Select registries.conf changes(/etc/containers/registries.conf. This file is generally changed via ICSP object changes. Only the following changes will cause no-reboot upgrades`

**Review by yuqi-zhang — COMMENTED** [review]
Overall lgtm, one nit:

File: docs/MachineConfigDaemon.md (hunk 1 above)
Comment (sinnykumari) [reply docs/MachineConfigDaemon.md]: fixed

**Review by yuqi-zhang — APPROVED** [review]
/lgtm

**Pull request merged**